rules/experimental/rbe/exec_properties.bzl

```diff
@@ -156,3 +242,105 @@ def merge_dicts(*dict_args):
     for dictionary in dict_args:
         result.update(dictionary)
     return result
+
+def _exec_property_sets_repository_impl(repository_ctx):
+    repository_ctx.file(
+        "BUILD",
+        content = "",
```

Comment:
Would it be relevant to have this BUILD file have a bzl_library target?

Reply:
I actually thought about it, but it seems that it works just fine without defining such a target.
What would be the reason to define this as a target?

Reply:
These targets need to be defined if a project depends on this rule from another rule and wants to, e.g., use skylib to generate docs.

Reply:
done